src/renderer/assets/styles/components/slider.scss

```diff
@@ -8,8 +8,12 @@
 
   &_wrapper {
     position: relative;
-    overflow: hidden;
+    overflow: scroll;
```

Comment:
There is a 1px scroll offset vertically, please use `overflow-x: hidden` and `overflow-y: scroll`